cli/aws_orbit/commands/deploy.py

```diff
@@ -318,6 +318,84 @@ def deploy_env(
         msg_ctx.progress(100)
 
 
+def deploy_base_images(
```

Comment:
is this needed right now? it's still using the old `remote.run` and `generate_bundle` functions. all of that is replaced by the toolkit

Reply:
I will remediate